packages/renderer/src/lib/preferences/docker-compat/PreferencesDockerCompatibilityContributions.svelte

```diff
@@ -115,12 +125,35 @@ function extractGroupItems(): void {
 
       const displayName = `${leftPart}: ${rightPart}`;
 
-      // add the property to the group
-      groupItem.properties.push({ ...property, displayName });
+      if (property.id) {
+        const newItem: PropertyWithDisplayName = { ...property, id: property.id, displayName };
+
+        // do we have a dynamic enum ?
+        if (property.enum) {
+          // get the enum from the context using the following key
+          // ${groupName}.DockerCompatibility.${property.id}
+
+          // get the enum from the context
+          const enumKey = `${name}.DockerCompatibility.${property.id}`;
+          const enumItems = globalContext.getValue<EnumItem[]>(enumKey);
+          if (enumItems) {
```

Comment:
As this may be set by an extension, should we check the type instead of assuming ? or it is okay to assert ?

Reply:
yes I thought the same before issuing the PR (to be more robust)
it'll be an addendum

Reply:
handled by a new commit